examples/install_simple/main.tf

```diff
@@ -37,7 +37,7 @@ provider "random" {
 
 module "forseti-install-simple" {
   source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.0.0"
+  version = "~> 5.1.0"
```

Comment:
Is this supposed to be 5.2.0, or is this to fix the tests?

Reply:
5.2.0 is not released yet, so that will fail. `~> 5.1.0` will use 5.2.0 once it is released. The alternative would be to update this example to point to GitHub instead of the TF repo, but I think the TF repo/versions is better to use.

Reply:
Ok thanks!